src/migrations/20190827125917-create-user.js

```diff
@@ -30,24 +44,34 @@ module.exports = {
       type: Sequelize.STRING,
       unique: true,
       validate: {
-        isEmail: true
+        isEmail: true,
+        notNull: true,
       }
     },
     phoneNumber: {
       allowNull: false,
       type: Sequelize.STRING,
       validate: {
-        not: ['[a-z]', 'i']
-      }
+        notNull: true,
+      },
+    },
+    isAdmin: {
```

Comment:
Please don't validate this field to be `not null` instead give it a default value of false. If the user wants to add an admin, he can specify that and if not, he doesn't have to

Reply:
ok